dependency_support/load_external.bzl

```diff
@@ -180,3 +180,57 @@ def load_external_repositories():
         strip_prefix = "ac_types-57d89634cb5034a241754f8f5347803213dabfca",
         build_file = "@com_google_xls//dependency_support/com_github_hlslibs_ac_types:bundled.BUILD.bazel",
     )
+
+    git_repository(
+        name = "platforms",
+        tag = "0.0.5",
+        remote = "https://github.com/bazelbuild/platforms.git",
+    )
+
+    http_archive(
+        name = "glpk",
+        build_file = "//bazel:glpk.BUILD",
+        sha256 = "4a1013eebb50f728fc601bdd833b0b2870333c3b3e5a816eeba921d95bec6f15",
+        url = "http://ftp.gnu.org/gnu/glpk/glpk-5.0.tar.gz",
+    )
+
+    http_archive(
+        name = "bliss",
+        build_file = "@com_google_ortools//bazel:bliss.BUILD",
+        patches = ["@com_google_ortools//bazel:bliss-0.73.patch"],
+        sha256 = "f57bf32804140cad58b1240b804e0dbd68f7e6bf67eba8e0c0fa3a62fd7f0f84",
+        url = "http://www.tcs.hut.fi/Software/bliss/bliss-0.73.zip",
+    )
+
+    new_git_repository(
+        name = "scip",
```

Comment:
SCIP uses the ZIB Academic License, which is also not compatible with Apache 2.0